unwetter/db.py

```diff
@@ -97,6 +98,10 @@ def query(severities, states, limit=50):
     else:
         filter['$or'] = [{'states': state} for state in states]
 
+    filter = {
+        'urgency': {'$in': urgencies},
```

Comment:
This line should be inserted after line 93 to be combined with the other filters